keyboards/eyeohdesigns/s50/keymaps/default/keymap.c

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2020 Eye Oh Designs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+//#include S50_H
+// Defines the keycodes used by our macros in process_record_user
+
+#define ______ KC_TRNS
+#define QWERTY 0
+#define RAISE 1
+#define NUMBRS 2
+enum custom_keycodes { QMKBEST = SAFE_RANGE, QMKURL };
```

Comment:
We recommend using enums for the layers, as it's easier to maintain, long term. 

```suggestion
enum layers {
    QWERTY,
    RAISE,
    NUMBRS,
};

enum custom_keycodes { 
    QMKBEST = SAFE_RANGE, 
    QMKURL,
};
```